app/src/main/java/de/westnordost/streetcomplete/quests/internet_access/AddInternetAccess.java

```diff
@@ -0,0 +1,52 @@
+package de.westnordost.streetcomplete.quests.internet_access;
+
+
+import android.os.Bundle;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+
+public class AddInternetAccess extends SimpleOverpassQuestType
+{
+	@Inject public AddInternetAccess(OverpassMapDataDao overpassServer) { super(overpassServer); }
+
+	@Override
+	protected String getTagFilters() {
+		return "nodes, ways, relations with " +
+				" amenity ~ restaurant|cafe|ice_cream|fast_food|bar|pub|biergarten|food_court|cinema|library" +
+				"  or" +
+				" tourism ~ hotel|guest_house|hostel|motel" +
+				" and !internet_access and name";
```

Comment:
Mmhh... Does it make sense to exclude an unofficial tag?

Reply:
Yes. There is already support for some bizarre tags like building:height to avoid surveying already collected data. See https://github.com/westnordost/StreetComplete/blob/master/app/src/main/java/de/westnordost/streetcomplete/quests/building_levels/AddBuildingLevels.java#L27

There is no point in asking people to resurvey, if it can be solved by documenting tag on wiki or by mechanical edit.